src/index.js

```diff
@@ -1,7 +1,8 @@
 // eslint-disable-next-line import/named
+// eslint-disable-next-line import/no-named-as-default-member
```

Comment:
- [ ] Kindly remove eslint disabler from your code.

> 🔧 Sometimes we may encounter challenges when using ESLint. Unexpected errors may arise which can be difficult to resolve quickly. If you come across an issue that is functioning properly but is still displaying an error message when uploaded to GitHub, I recommend disabling it in the .eslintrc.json file. This can help to prevent any confusion or delays in the development process. Let me know if you need any assistance.